src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageItemListViewModel.cs

```diff
@@ -196,17 +196,17 @@ public string LatestVersionToolTip
             }
         }
 
-        private bool _selected;
+        private bool _isSelected;
```

Comment:
nit: I don't see any value added to the Project lists bug with this refactoring, but several files/changes are needed to do it.

Given that we are refactoring these classes soon anyway, I'm not sure it's a great idea to rename variables.

Reply:
The internal variable name change is so that it matches the name of the Property, for clarity.

The property was changed so it adheres to and implements the ISelectableItem interface which allows all objects of that type to be treated equally. This is the basis under which the AutomationPeer can be reused so all items have a "IsSelected" property.

Reply:
The reason doesn't quite make sense to me... this VM is completely independent of the AP; however, the change is acceptable